### ISSUE TITLE
Thrown "Debug Failure. Invalid container kind. SyntaxKind: InterfaceDeclaration" for readonly members

### ISSUE THREAD

**Issue by ulrichb** [microsoft/monaco-editor#1501]
**monaco-editor version:** 0.17.1
**Browser:** Chrome / Firefox
**OS:**  Win

**Steps or JS usage snippet reproducing the issue:**

Go to the [Monaco editor playground](https://microsoft.github.io/monaco-editor/playground.html), use the following snippet and open the F12 tools.

```ts
monaco.editor.create(document.getElementById("container"), {
	value: `interface SomeInterface {
    readonly prop: string;
};
`, language: "typescript"
});
```
When navigating the cursor before the first character of `readonly` (line 2, char 5), the following error appears in the error console:

![image](https://user-images.githubusercontent.com/388796/60451038-714e6280-9c2b-11e9-8b8c-a4a012bc114a.png)

**Comment by ulrichb** [comment]
This has been accepted as bug in the TS repo (see link above). => Waiting for a TS version with a fix.

**Comment by alexdima** [comment]
I believe this has been fixed with the monaco editor 0.18.x
